conf/cert/two_side_server.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
I don't think it's a good idea to have a few built in self-signed certs, they should only be used for testing.